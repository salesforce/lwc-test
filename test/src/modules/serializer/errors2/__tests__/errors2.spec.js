/*
 * Copyright (c) 2023, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { createElement } from 'lwc';
import Errors2 from '../errors2';

it('serializes component that may cause errors in serializer, part 2', async () => {
    const elm = createElement('serializer-component', { is: Errors2 });
    document.body.appendChild(elm);

    expect(elm).toMatchSnapshot();
});
