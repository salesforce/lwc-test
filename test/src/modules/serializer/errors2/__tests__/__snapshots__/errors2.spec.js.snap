// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializes component that may cause errors in serializer, part 2 1`] = `
<serializer-component>
  #shadow-root(open)
    <div />
</serializer-component>
`;
